app/services/db-export/send-to-s3-bucket.service.js

```diff
@@ -47,7 +74,7 @@ async function _uploadToBucket (params, fileName) {
 
     return true
   } catch (error) {
-    return false
+    throw new Error()
```

Comment:
This doesn't need to be a `try...catch` block now that we're no longer handling the error in this function. The calling function has it wrapped in a `try...catch` so if `s3Client.send()` throws an error then it will be caught by that; we don't need to catch it and then throw a new error.